clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesKind.java

```diff
@@ -131,24 +139,30 @@
   @Getter private final boolean isRegistered;
 
   private KubernetesKind(
-      String name,
-      KubernetesApiGroup apiGroup,
-      String alias,
+      @Nonnull String name,
+      @Nonnull KubernetesApiGroup apiGroup,
+      @Nullable String alias,
       boolean isNamespaced,
       boolean hasClusterRelationship) {
     this(name, apiGroup, alias, isNamespaced, hasClusterRelationship, false, true);
   }
 
   private KubernetesKind(
-      String name,
-      KubernetesApiGroup apiGroup,
-      String alias,
+      @Nonnull String name,
+      @Nonnull KubernetesApiGroup apiGroup,
+      @Nullable String alias,
       boolean isNamespaced,
       boolean hasClusterRelationship,
       boolean isDynamic,
       boolean isRegistered) {
     this.name = name;
+    this.lcName = name.toLowerCase();
```

Comment:
I probably just missed it but where is `lcName` being used?

Reply:
It's used above in the `@EqualsAndHashCode.Include @Nonnull private final String lcName;`  I wanted case-insensitive equality so decided to store the lower-case version of the name as its own variable and use that in the auto-generated equality check.  It's not directly exposed to consumers of the class anywhere though.

Reply:
Oh cool I didn't fully grok @EqualsAndHashCode.Include until I read about it just now, nice!!